tests/ttnn/unit_tests/gtests/tensor/test_vector_conversion.cpp

```diff
@@ -0,0 +1,116 @@
+// SPDX-FileCopyrightText: (c) 2024 Tenstorrent AI ULC
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#include <gtest/gtest.h>
+#include <gmock/gmock.h>
+#include <algorithm>
+#include <cstdint>
+
+#include "ttnn/tensor/tensor.hpp"
+#include "ttnn/tensor/xtensor/conversion_utils.hpp"
+
+namespace ttnn {
+namespace {
+
+using ::testing::Eq;
+using ::testing::Pointwise;
+using ::ttnn::experimental::xtensor::from_vector;
+using ::ttnn::experimental::xtensor::to_vector;
+
+const std::vector<ttnn::SimpleShape>& GetShapesForTest() {
+    static auto* shapes = new std::vector<ttnn::SimpleShape>{
```

Comment:
don't need to use new here.

Reply:
We don't, just muscle memory to never use static storage for types with non-trivial dtor:)

Reply:
this should be corrected, there is really no need for `new` here

Reply:
Hmm wait - what is the issue exactly? The pattern is just so that we don't have a static variable to run a non-trivial dtor. See https://google.github.io/styleguide/cppguide.html#Static_and_Global_Variables

Here of course it does not matter and I can just return a copy. No strong opinion, but was wondering where are you coming from:)